src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,100 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
 import './FindMovie.scss';
+import { MovieData } from '../../types/MovieData';
+import { MovieCard } from '../MovieCard';
+import { DEFAULT_IMG } from '../../utils';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  onAddMovieBtn: (newMovie: Movie) => void;
+  thisMovieExist: boolean
+};
+
+export const FindMovie: React.FC<Props> = ({
+  onAddMovieBtn, thisMovieExist,
+}) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [onError, setOnError] = useState(false);
+
+  const createNewMovie = (foundMovieData:MovieData | null) => {
+    if (foundMovieData !== null) {
+      const {
+        Poster,
+        Title,
+        Plot,
+        imdbID,
+      } = foundMovieData;
+
+      const poster = Poster === 'N/A'
+        ? DEFAULT_IMG
+        : Poster;
+
+      const newMovieItem = {
+        title: Title,
+        description: Plot,
+        imgUrl: poster,
+        imdbUrl: `https://www.imdb.com/title/${imdbID}`,
+        imdbId: imdbID,
+      };
+
+      setMovie(newMovieItem);
+      setIsLoading(false);
+    }
+  };
+
+  const loadMovie = async (searchQuery:string) => {
+    try {
+      const responce = await getMovie(searchQuery);
+
+      if ('Error' in responce) {
+        throw new Error();
+      } else {
+        createNewMovie(responce);
+        setOnError(false);
+      }
+    } catch (error) {
+      setOnError(true);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const onSubmitHandler = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setIsLoading(true);
+    loadMovie(query);
+  };
+
+  const onInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (onError && query === e.target.value) {
+      setOnError(true);
+    } else {
+      setOnError(false);
+    }
```

Comment:
```suggestion
      setOnError(onError && query === e.target.value);
```

Reply:
I need to set it fasle, not only check

Reply:
So `onError && query === e.target.value` equal to true or false. 
`setOnError(onError && query === e.target.value);` will work